website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+[WARN]  identity: entity "entity-case" with namespace ID "root" duplicates 2 others: id=b0141be5-3f03-a1c7-a57b-02f045d04426 force_deduplication="would not rename"
+[WARN]  identity: entity "entity-cAsE" with namespace ID "sYMXY" duplicates 2 others: id=95f0743b-a1d5-26da-b4ef-a50490da0787 force_deduplication="would rename to entity-cAsE-95f0743b-a1d5-26da-b4ef-a50490da0787"
+[WARN]  identity: entity "entity-case" with namespace ID "sYMXY" duplicates 2 others: id=9d3be96f-490a-9625-3118-896bd2b3a5f3 force_deduplication="would rename to entity-case-9d3be96f-490a-9625-3118-896bd2b3a5f3"
+[WARN]  identity: entity "entity-case" with namespace ID "sYMXY" duplicates 2 others: id=d82231d2-3716-6b9c-e80d-7d09c0409739 force_deduplication="would not rename"
+[WARN]  identity: end of entity duplicates
+[WARN]  identity: 2 group duplicates found
+[WARN]  identity: 2 group duplicates found
+[WARN]  identity: group "group-case" with namespace ID "root" duplicates 2 others: id=8ad26e0c-8cf6-5b67-7c77-6571fa374f34 force_deduplication="would rename to group-case-8ad26e0c-8cf6-5b67-7c77-6571fa374f34"
+[WARN]  identity: group "group-cAsE" with namespace ID "root" duplicates 2 others: id=9fe86ea0-f80c-1131-5be1-1d6e3b70237f force_deduplication="would rename to group-cAsE-9fe86ea0-f80c-1131-5be1-1d6e3b70237f"
+[WARN]  identity: group "group-case" with namespace ID "root" duplicates 2 others: id=32dd070c-c1f8-c796-9a71-15887014b813 force_deduplication="would not rename"
+[WARN]  identity: group "group-case" with namespace ID "sYMXY" duplicates 2 others: id=8aaeff7e-7343-c883-1e0c-c5c9968f75a5 force_deduplication="would rename to group-case-8aaeff7e-7343-c883-1e0c-c5c9968f75a5"
+[WARN]  identity: group "group-case" with namespace ID "sYMXY" duplicates 2 others: id=f11277b3-d985-4d72-d2e9-9c8c6c0db02c force_deduplication="would rename to group-case-f11277b3-d985-4d72-d2e9-9c8c6c0db02c"
+[WARN]  identity: group "group-cAsE" with namespace ID "sYMXY" duplicates 2 others: id=7c753d07-b0d9-e13b-6184-48247b8f7504 force_deduplication="would not rename"
+[WARN]  identity: end of group duplicates
+```
+
+Entity and group duplicates are reported as in the examples above. The
+resolution steps are the same for both.
+
+Entities or groups may exist with duplicate names. These might be exact matches,
+or may differ in case such as `Admin` and `admin`. In either case, the
+resolution is the same.
+
+Note that names are only duplicates if they are in the same namespace.
+
+Possible unexpected behavior caused by these types of duplicates includes:
+ * Only one of them will be returned when looking up or listing by name
+ * Before Vault 1.19.0, which one is returned might vary depending on which
+   server is hit or even after a seal and unseal on the same server
+ * Lookup by ID _will_ work for all duplicates
```

Comment:
Why is "will" italicized here?

Reply:
Originally because this is a list of "unexpected behavior" and this item on it's own is not unexpected it's more meant to clarify since the other forms of lookup don't work as expected. I think it might be better just to leave it out though.